client/pkg/pynuoadmin.py

```diff
@@ -30,53 +30,25 @@ def prereqs(self):
         # We need nuodb to get nuokeymanager.jar
         return ['nuodb']
 
-    def download(self):
-        # Find the latest release
-        pypi = PyPIMetadata('pynuoadmin')
-
-        self.setversion(pypi.version)
-
-        self._file = Artifact(self.name, 'pynuoadmin.tar.gz',
-                              pypi.pkgurl, chksum=pypi.pkgchksum)
-        self._file.update()
-
-        # Grab the latest version of argcomplete
-        acpypi = PyPIMetadata('argcomplete')
-        self._ac_version = acpypi.version
-        self._ac_file = Artifact('argcomplete',
-                                 'argcomplete-{}.tar.gz'.format(self._ac_version),
-                                 acpypi.pkgurl,
-                                 chksum=acpypi.pkgchksum)
-        self._ac_file.update()
-
     def unpack(self):
         rmdir(self.pkgroot)
         mkdir(self.pkgroot)
-        unpack_file(self._file.path, self.pkgroot)
-        self.stage.basedir = os.path.join(self.pkgroot, 'pynuoadmin-{}'.format(self.stage.version))
-
-        unpack_file(self._ac_file.path, self.pkgroot)
-        self.ac_basedir = os.path.join(self.pkgroot, 'argcomplete-{}'.format(self._ac_version))
+        run(['pip', 'install', self.__PKGNAME, '-t', self.pkgroot])
```

Comment:
Please use `import sys` and change this to:
```suggestion
        run([sys.executable, 'pip', 'install', self.__PKGNAME, '-t', self.pkgroot])
```